examples/convert_group_studies.py

```diff
@@ -63,13 +65,15 @@
 for subject_id in subject_ids:
     subject = 'sub%03d' % subject_id
     for run in runs:
-        raw_file = op.join(data_path, repo, subject, 'MEG',
-                           'run_%02d_raw.fif' % run)
-
-        # Make it BIDS compatible
-        raw_to_bids(subject_id='%02d' % subject_id, session_id='01', run=run,
-                    task='VisualFaces', raw_file=raw_file,
-                    event_id=event_id, output_path=output_path, overwrite=True)
+        raw_fname = op.join(data_path, repo, subject, 'MEG',
+                            'run_%02d_raw.fif' % run)
+        raw = mne.io.read_raw_fif(raw_fname)
```

Comment:
is there a reason to create a Raw object as opposed to just providing the path above

Reply:
Two reasons for me:

1. `raw` object can be manipulated. If you just give a filename, users will start asking us for functions or new arguments for every small change they need to do. Instead, they could just change the `raw` object with one or two lines of `numpy` code when necessary
2. The `read_raw` object is likely to grow larger with more arguments as we incorporate more file types. It just feels to me that a general `raw` reader should be kept separate from the issue of writing out a BIDS compatible folder.